From 3dc824599c2a2e39c598a4d57d6ba42a3feea78e Mon Sep 17 00:00:00 2001
From: Thomas Bechtold <tbechtold@suse.com>
Date: Sat, 29 Jul 2017 08:31:33 +0200
Subject: [PATCH] Fix man page build

Ia3b5ee2b6aaf17781d3de7546a9477f1c928092f moved cinder.rst from the
doc/source/man directory into doc/source/cli, so we need to adjust
the path in conf.py to avoid issues when running:

python setup.py build_sphinx -b man

Change-Id: I32ca20e0e184b11c68c9b81a9ff3d1247330ade4
---
 doc/source/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 1af8db5..2efc731 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -112,7 +112,7 @@ pygments_style = 'sphinx'
 
 
 man_pages = [
-    ('man/cinder', 'cinder', u'Client for OpenStack Block Storage API',
+    ('cli/details', 'cinder', u'Client for OpenStack Block Storage API',
      [u'OpenStack Contributors'], 1),
 ]
 # -- Options for HTML output --------------------------------------------------
-- 
2.13.3

